docker/test/integration/minifi/test/__init__.py

```diff
@@ -213,24 +245,75 @@ def __init__(self, expected_content):
         self.valid = False
         self.expected_content = expected_content
 
-    def validate(self):
+    def validate(self, dir=''):
 
-        if self.valid:
-            return True
+        # TODO: Why did we do this btw?
```

Comment:
To avoid validating sg. twice. 
In case a validator already passed, we are happy. 

Not sure if that really occurs or not, I guess not, so I don't mind this to be removed.

Reply:
Will remove it, since I run validate many times with different dirs